fearless/Common/Operation/StorageDecodingOperation.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import FearlessUtils
+import RobinHood
+
+enum StorageDecodingOperationError: Error {
+    case missingRequiredParams
+    case invalidStoragePath
+}
+
+final class StorageDecodingOperation<T: Decodable>: BaseOperation<T> {
+    var data: Data?
```

Comment:
let?

Reply:
This dependency can come from another operation, so we need an opportunity to update this in config closure